backend/qemu.pm

```diff
@@ -711,10 +715,17 @@ sub start_qemu {
                 }
             }
             else {
+                my $diskmodel;
+                if (defined $vars->{"HDDMODEL_$i"}) {
+                    $diskmodel = $vars->{"HDDMODEL_$i"};
+                }
+                else {
```

Comment:
I would write this as 

```
my $diskmodel = $vars->{"HDDMODEL_$i"} || $vars->{HDDMODEL};
```

There are various other examples of such a usage in qemu.pm. But I won't block merging for this